AttitudeManager/ControlAlgorithms/Src/AM_FixedControl.cpp

```diff
@@ -9,7 +12,58 @@ std::vector<ActuatorOutput> FixedControl::runControlsAlgorithm(
     // interpret instructions
     // calculate pids
     // mix pids
-    // return output
+   // Get current attitude from sensorfusion 
+   
+    SFOutput_t current_attitude; 
+    
+    // Compute target values
+    float target_heading = instructions.heading;
+    float target_throttle = instructions.speed; 
+    float target_pitch = instructions.x_dir * MAX_PITCH_ANGLE;
+    float target_bank = instructions.y_dir * MAX_BANK_ANGLE;
+     
+    // Adjust heading difference for the PID. Can be reworked 
+    // if (targetHeading <= (-180)) {
+    //     targetHeading += 180;
+    // } else if (targetHeading > 180){
+    //     targetHeading -=360;
+    // }                                      
```

Comment:
What is this for, and why is it commented out? You should either delete it or use it I think

Reply:
Before, I was computing the desired heading by finding the difference between the current and desired heading given to AM. This would ensure the title is relative to a particular axis. Instead, I moved this into our PID for yaw, which computes the target and current heading.